src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -4855,6 +4856,12 @@ void Parser::parse_and_visit_import(
           QLJS_PARSER_UNIMPLEMENTED_IF_NOT_TOKEN(Token_Type::right_paren);
           this->skip();
         } else {
+          // import type a = b; // Invalid.
+          if (declared_variable_kind == Variable_Kind::_import_type) {
```

Comment:
Nit: This comment belongs inside the `if`.
```suggestion
          if (declared_variable_kind == Variable_Kind::_import_type) {
            // import type a = b; // Invalid.
```

Reply:
done